### ISSUE TITLE
Remove deprecated Sass from project

### ISSUE THREAD

**Issue by shiny-eel** [kblincoe/VisualGit_SE701_3#91]
### Problem
In the **stylesheets** directory, there are two Sass (.scss) files. These are outdated. When compiled locally, they overwrite more recent changes made to the **styles.css** file.

It has been determined that changes should be made directly to the styles.css file, and that the Sass files are no longer useful.

The README.md gives outdated information, saying that the Sass files are still to be used to create the style.css file.

**Steps to Reproduce**
1. Fork the VisualGit Repo for development
2. Edit the _stylesheets/styles.scss_ or _stylesheets/variables.scss_ files
3. Compile the .scss files using **node_sass** or **Ruby**.
4. Build the app, and see the failed styling.

### Actions
EDIT- The README should be edited to reflect the current way of working (i.e. without overwriting the styles.css using the Sass). Also, the Sass files should be removed or simply placed in an isolated directory where they do not affect the build.

**Comment by shiny-eel** [comment]
@softeng-701

**Comment by softeng-701** [comment]
@shiny-eel The Sass files no longer have any function? You don't foresee simply removing them causing any issues?

**Comment by shiny-eel** [comment]
@softeng-701 You're right - possibly the best solution is simply putting them aside in an unused directory within the repo.

We have tested removing them and running VisualGit, and have found no differences in functionality or appearance so far. In the last lecture on Friday, @kblincoe also seemed to acknowledge that the Sass files were no longer as up to date as the css file (when a student came up to talk about the issues experienced so far with the assignment).

_At the minimum_, the README.md should be changed to help prevent future developers accidentally overwriting the styles.css file by compiling the Sass.

**Comment by softeng-701** [comment]
@shiny-eel This all sounds reasonable. The assignee just needs to be careful that new issues aren't introduced by changing this. Thanks!
